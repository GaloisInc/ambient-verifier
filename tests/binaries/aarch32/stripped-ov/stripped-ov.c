#include "ambient_assert.h"

int f() {
  return 1;
}

int main() {
  ambient_assert(f() == 0);
  return 0;
}
