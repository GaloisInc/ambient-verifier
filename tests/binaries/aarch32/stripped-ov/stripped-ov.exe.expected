successful: 0
failed: 0
overrideDir: tests/binaries/x86_64/stripped-ov/overrides
entryPointAddr: 0x10090
